java/cuvs-java/src/test/java/com/nvidia/cuvs/CagraRandomizedIT.java

```diff
@@ -45,10 +45,10 @@ public void setup() {
   @Test
   public void testResultsTopKWithRandomValues() throws Throwable {
     boolean useNativeMemoryDatasets[] = {true, false};
-	  for (int i = 0; i < 100; i++) {
-	    for (boolean use: useNativeMemoryDatasets) {
-		     tmpResultsTopKWithRandomValues(use);
-	    }
+    for (int i = 0; i < 100; i++) {
```

Comment:
There are a lot of whitespace fixes that are unrelated to this PR. Should we deal with linting, whitespace fixes etc in an entirely separate PR?